packaging/osx/package-osx.sh

```diff
@@ -12,26 +12,65 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
-source "$DIR/../../scripts/common/_common.sh"
+help(){
+    echo "Usage: $0 [--version <pkg version>] [--input <input directory>] [--output <output pkg>] [--help]"
+    echo ""
+    echo "Options:"
+    echo "  --version <pkg version>          Specify a version for the package."
+    echo "  --input <input directory>        Package the entire contents of the directory tree."
+    echo "  --output <output pkg>            The path to which the package will be written."
+    exit 1
+}
+
+while [[ $# > 0 ]]; do
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    case $lowerI in
+        -v|--version)
+            DOTNET_CLI_VERSION=$2
+            shift
+            ;;
+        -o|--output)
+            OUTPUT_PKG=$2
+            shift
+            ;;
+        -i|--input)
+            INPUT_DIR=$2
+            shift
+            ;;
+        --help)
+            help
+            ;;
+        *)
+            break
+            ;;
+    esac
+    shift
+done
 
 if [ -z "$DOTNET_CLI_VERSION" ]; then
-    echo "Provide a version number (DOTNET_CLI_VERSION) $DOTNET_CLI_VERSION" && exit 1
+    echo "Provide a version number. Missing option '--version'" && help
+fi
+
+if [ -z "$OUTPUT_PKG" ]; then
+    echo "Provide an output pkg. Missing option '--output'" && help
 fi
 
-STAGE2_DIR=$REPOROOT/artifacts/$RID/stage2
+if [ -z "$INPUT_DIR" ]; then
+    echo "Provide an inout directory. Missing option '--input'" && help
+fi
 
-if [ ! -d "$STAGE2_DIR" ]; then
-    echo "Missing stage2 output in $STAGE2_DIR" 1>&2
+if [ ! -d "$INPUT_DIR" ]; then
+    echo "Missing input directory - $INPUT_DIR" 1>&2
```

Comment:
Should you change this to say that the path is not a directory? Instead that it is missing?